<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="btn_close">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>157</y>
     <width>94</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/ok.png);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="displaylabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>89</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/message.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pixmap.qrc">:/pixmap/image/message.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>55</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务提醒：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>156</y>
     <width>94</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/quit.png);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>195</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/dialog-bg.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>btn_close</zorder>
  <zorder>okButton</zorder>
  <zorder>displaylabel</zorder>
  <zorder>label</zorder>
  <zorder>closeButton</zorder>
 </widget>
 <resources>
  <include location="pixmap.qrc"/>
 </resources>
 <connections/>
</ui>
